webapp/src/components/backstage/playbook_list.tsx

```diff
@@ -331,18 +346,23 @@ const Button = styled.button`
     }
 `;
 
-const NoContentPage = (props: { onNewPlaybook: () => void }) => {
+const NoContentPage = (props: { onNewPlaybook: () => void, canCreatePlaybooks: boolean }) => {
     return (
         <Container>
             <Title>{'What is a Playbook?'}</Title>
             <Description>{'A playbook is a workflow template which must be created before an incident occurs. It defines the stages and tasks associated with an incident, as well as who can use playbook to start an incident.'}</Description>
-            <Button
-                className='mt-6'
-                onClick={() => props.onNewPlaybook()}
-            >
-                <i className='icon-plus mr-2'/>
-                {'New Playbook'}
-            </Button>
+            { props.canCreatePlaybooks &&
+                <Button
+                    className='mt-6'
+                    onClick={() => props.onNewPlaybook()}
+                >
+                    <i className='icon-plus mr-2'/>
+                    {'New Playbook'}
+                </Button>
+            }
+            { !props.canCreatePlaybooks &&
+            <DescriptionWarn>{'There are no playbooks and you do not have permissions to create playbooks on this server.'}</DescriptionWarn>
```

Comment:
```suggestion
            <DescriptionWarn>{'You don't have permission to view and create playbooks on this server.'}</DescriptionWarn>
```
Is this strictly accurate? There may surely be playbooks but the viewer just can't see and create them because of permissions?

Reply:
@justinegeffen 
Ya, there might be playbooks they can't view. 
> You don't have permission to view and create playbooks on this server.

Is not accurate because they have permission to view playbooks, but there just are none that they qualify to view at the moment. 

Perhaps:
```
There are no playbooks to view. You don't have permission create playbooks on this server.
```

Reply:
Nice suggestion, thanks! I've updated my suggestion. :)